components/inputs/docs/input-select-styles.md

```diff
@@ -48,60 +48,159 @@ Toggling progressive disclosure is OK
 
 ## Applying styles to native select elements
 
-Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class.
+Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class. The styles account for several CSS pseudo-classes, including `disabled`, `focus` and `hover`.
 
-Note: in order for RTL to function correctly, make sure your component uses the `RtlMixin`.
+<!-- docs: demo code name:d2l-test-input-select -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+  class TestInputSelect extends LitElement {
+  static get styles() {
+    return [ selectStyles ];
+  }
+
+  render() {
+    return html`
+      <select
+        aria-label="Choose a dinosaur:"
+        class="d2l-input-select">
+        <option>Tyrannosaurus</option>
+        <option>Velociraptor</option>
+        <option>Deinonychus</option>
+      </select>
+    `;
+  }
+
+  }
+  customElements.define('d2l-test-input-select', TestInputSelect);
+</script>
+<d2l-test-input-select></d2l-test-input-select>
+```
+
+## Building a Custom Select Component
```

Comment:
Hmm I feel like `RtlMixin` and `SkeletonMixin` should have separate documentation and perhaps just get called out here as something to be considered and then with a link to the docs.
Since they aren't in Daylight officially yet, you could point at their GitHub docs [here](https://github.com/BrightspaceUI/core/tree/main/components/skeleton#skeletizing-custom-elements-with-skeletonmixin) and [here](https://github.com/BrightspaceUI/core/tree/main/mixins/rtl#rtlmixin)

Reply:
So both those sections do link to their GitHub docs. I'm removing the `RtlMixin` section based on Dave's comment, but was planning on leaving the `SkeletonMixin` section. I figure it's still worth keeping and showing off what it looks like, while still linking to the doc, or would you prefer no code showing up?

Reply:
After talking in the meeting, I'll be removing the section and switching to just making a reference to the `SkeletonMixin` earlier on.